src/core/ui.js

```diff
@@ -131,6 +133,35 @@ function createWarnButton(butName, arr, title) {
   return button;
 }
 
+function focusTrapper(e, firstFocussable, lastFocussable) {
```

Comment:
This is better, but it's still not quite right... as I mentioned previously, is should be possible to use "focusTrapper" independently of anything else... I suggest doing this: 

 1. copy/paste the exact code from [https://hiddedevries.nl](https://hiddedevries.nl/en/blog/2017-01-29-using-javascript-to-trap-focus-in-an-element) to core/utils.js without modifying it and `export function trapFocus(element){}` (remove the namespace, it's not used).
 1. import that function into here. 
 1. Only when the menu opens, call `trapFocus(menu);`
 1. When to menu closes, see if `trapFocus()` knows to release the focus.

Reply:
We should keep related functions in a relevant module if possible. I would keep that function here as we only use that function in this module.

Reply:
My idea is to use it for all the dialogs, as they all require the same thing... let me check where they come from.